index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="main">
+      <div id="title">
+        <h1>Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </div>
+      <div id="image-div">
+        <img src="tribute-page-main-image.jpg" alt="The image" id="image">
+        <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+      </div>
+      <div id="tribute-info">
```

Comment:
It will be good to use HTML semantic tags over div s

Reply:
Great job @Samuellekpor 🤝

- [ ] You may kindly explain `why` you suggested the use of semantic tags over `divs`.